platform/stm32/export/aux/usart_bus.hpp

```diff
@@ -307,21 +327,17 @@ void usart_bus<dev>::set_rx(uint8_t *rx, size_t size)
 template<usart_device dev>
 void usart_bus<dev>::set_tx(size_t size, uint8_t fill_byte)
 {
-    if (!inited()) {
-        return;
-    }
+    // TODO: implement
+    ecl_assert_msg(0, "Not implemeted!");
```

Comment:
Static assert will hit, even if there is no function call immediately after template instance is created.